acestep/ui/gradio/events/training/preprocess.py

```
@@ -0,0 +1,244 @@
+    if not output_dir or not output_dir.strip():
+        return "❌ Please enter an output directory."
+
+    if dit_handler is None or dit_handler.model is None:
+        return "❌ Model not initialized. Please initialize the service first."
+
+    def progress_callback(msg):
+        if progress:
+            try:
+                progress(msg)
+            except Exception:
+                pass
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)